tests/virt_autotest/login_console.pm

```diff
@@ -120,6 +120,12 @@ sub login_to_console {
             #send key 'up' to stop grub timer counting down, to be more robust to select xen
             send_key 'up';
             save_screenshot;
+            #If sol console is stuck, reconnect it and resend key up
+            if (check_screen 'virttest-bootmenu-xen-kernel-count-down', 1) {
+                reset_consoles;
+                select_console 'sol', await_console => 0;
+                send_key 'up';
+            }
```

Comment:
I think your change may have impact on other archs, for example, arm64 as well. It needs to be paid attention.